webpack.config.js

```diff
@@ -0,0 +1,42 @@
+var path = require('path');
+var SRC_DIR = path.join(__dirname, '/client/src');
+var DIST_DIR = path.join(__dirname, '/client/dist');
+
+module.exports = {
+  entry: `${SRC_DIR}/index.jsx`,
+  output: {
+    filename: 'workspaces.js',
+    path: DIST_DIR
+  },
+  mode: 'development',
+  module: {
+    rules: [
+      {
+        test : /\.jsx?/,
+        exclude: /node_modules/,
+        loader: 'babel-loader',
+        options: {
+          presets: ['@babel/preset-react']
+        }
+      },
+      {
+        test: /\.s[ac]ss$/i,
+        exclude: /node_modules/,
+        use: [
+          'style-loader',
+          'css-loader',
+          'sass-loader',
+        ],
+      },
+      {
+        test: /\.(png|jpg|gif)$/,
+        use: [{
+            loader: "file-loader",
+            options: {
+              name: '[name].[ext]'
+            }
+        }]
```

Comment:
The current config works but another approach you might consider is not bundling your images and rather serve them statically and use a url as the image.src in the component. This might help with performance (will get back to you on that)

Reply:
Hmm, ok, I'll look into that too, thanks.